frontend/src/features/login/LoginPage.tsx

```diff
@@ -115,8 +116,10 @@ export const LoginPage = (): JSX.Element => {
 
   const handleSendOtp = async ({ email }: LoginFormInputs) => {
     const trimmedEmail = email.trim()
-    await sendLoginOtp(trimmedEmail)
-    return setEmail(trimmedEmail)
+    await sendLoginOtp(trimmedEmail).then(({ otpPrefix }) => {
+      setOtpPrefix(otpPrefix)
```

Comment:
Just noting: during deployment, if/when receiving a response from an old server, this would be `undefined` rather than a string. It looks harmless in the code and its uses, and would always be a string as soon as deployment is over (few minutes tops).

Reply:
Just to note, the OTP field should look like the following when the UI is receiving a response from an old server (simulated with React DevTools):

<img width="1512" alt="Screenshot 2023-02-24 at 9 21 58 AM" src="https://user-images.githubusercontent.com/37061143/221069169-272e985a-209f-4612-97e1-f095de88008d.png">

Reply:
Sorry, as discussed, the screenshot is wrong :') there will be a ' -' string in the grey prefix part.

Reply:
I wonder if it was not doubly-wrong even? 😅🤔 

As-in, the string interpolation of `undefined` and the dash would probably have shown the string `undefined -`, no?

```
> let foo;
undefined

> `${foo} -`
'undefined -'
```

Reply:
oh yea 🤦‍♀️ but its okay I think it should be fine now with the latest commit [6480f5f](https://github.com/opengovsg/FormSG/pull/5828/commits/6480f5fa3ccba88ef24a9988cecb16813a01ec8d)! let me test it out

Reply:
Seems to work on my end!